### ISSUE TITLE
[FastPass] missing labels for input fields

### ISSUE THREAD

**Issue by vyncent-t** [kubernetes-sigs/headlamp#4344]
## Describe the bug

There are currently some FastPass issues for one of our custom input components. There are 6 total issues relating to missing labels for the inputs used. 

Below are screenshots of the FastPass Scans and the places in the UI where the issues occur.

FastPass scan
<img width="1638" height="630" alt="Image" src="https://github.com/user-attachments/assets/67422917-e684-4b76-9d30-ed4cc2828b0c" />

Inputs that need fixes
<img width="796" height="536" alt="Image" src="https://github.com/user-attachments/assets/c85937f6-0e91-45fd-ba05-a71d683c2a55" />

## To Reproduce
Steps to reproduce the bug:
1. Launch FastPass in a different window
2. Navigate to the Secrets list view
3. Navigate to any Secrets resource view
4. Run FastPass on the opened Secrets resource
5. Note the missing inputs

## Are you able to fix this issue?
Yes
